src/app/breadcrumbs/breadcrumbs.component.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('BreadcrumbsComponent', () => {
     expect(breadcrumbs.length).toBe(3);
     expectBreadcrumb(breadcrumbs[0], 'home.breadcrumbs', '/');
     expectBreadcrumb(breadcrumbs[1], 'bc 1', '/example.com');
-    expectBreadcrumb(breadcrumbs[2].query(By.css('.text-truncate')), 'bc 2', null);
+   expectBreadcrumb(breadcrumbs[2].query(By.css('.text-truncate')), ' bc 2', null);
```

Comment:
Hi @tdonohue,

I have finished with the suggestions and uploaded the changes.

Thank you very much for your comments.

Regards!

Reply:
@jtimal  i see this change is still present, can you undo it? thanks

Reply:
hi @atarix83 I have already undone the change

Reply:
@jtimal : The problem was that when you undid the change, the alignment in that file got accidentally messed up.  I just fixed it by pushing a commit to your branch: https://github.com/DSpace/dspace-angular/pull/1957/commits/8c8df56c1a197732b5d8b031a79f6519fdf4dd46

With that fix, the changes to `breadcrumbs.component.spec.ts` have been fully reverted, so that it no longer appears in the "Files changed" tab at the top of this PR.

Reply:
@tdonohue ok, thank you very much and an apology for this.